doc/webui/user_attributes.rst

```diff
@@ -0,0 +1,30 @@
+.. _user_attributes:
+
+Additional user attributes
+--------------------------
+
+.. index:: User Attributes, Additional User Attributes
+
+Since version 3.6 privacyIDEA allows to manage additional internal attributes for
```

Comment:
Since this appears in the documentation of version 3.6, we may drop the explicit reference here.

Reply:
We have this information in a lot of pollicies. 
Imagine looking at the docs of version 3.8. This way you would not know, if this feature was added in 3.8 or in an older version. This helps, when you look at the docs  to know, if you are looking at a new or at an old, established feature.